### ISSUE TITLE
vertical_parameter_alignment_on_call shouldn't trigger after blocks

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1630]
```swift
// shouldn't trigger
UIView.animate(withDuration: 0.4, animations: {
    blurredImageView.alpha = 1
}, completion: { _ in
    self.hideLoading()
})
```

**Comment by ileitch** [comment]
I personally quite like the new format of 
```
UIView.animate(
    withDuration: 0.4,
    animations: {
        blurredImageView.alpha = 1
    },
    completion: { _ in
        self.hideLoading()
    }
)
```

However, what is problematic is last argument closures:
```
transition(
    from: fromController,
    to: toController,
    duration: 0.75,
    options: options.union([.curveEaseInOut]),
    animations: {
        toController.view.snp.makeConstraints { make in
            make.edges.equalToSuperview()
        }
    }
) { _ in
    fromController.removeFromParentViewController()
    toController.didMove(toParentViewController: self)
    self.changeRightBarButtonItem(to: newRightButton)
}
```

If I do indent the `{ _ in` line to satisfy the rule, the `opening_brace` then triggers for the same line. I could explicitly label the last argument, but I feel the lint rule should change rather than my code in this instance.

**Comment by ileitch** [comment]
Here's another erroneous case:
```swift
collectionView?.performBatchUpdates({ [weak self] in
        guard let strongSelf = self else { return }
        strongSelf.performChanges()
    },
    completion: { [weak self] (_) in
        guard let strongSelf = self else { return }

        strongSelf.delegate?.collectionViewBatchUpdaterDidCompleteBatchUpdates?(strongSelf)
    }
)
```

As in my previous example, moving the first closure to a newline triggers the `opening_brace` rule.

An argument label cannot be added to the first closure argument as this is a UIKit function which is exposed in Swift as `func performBatchUpdates(_ updates: ...`

**Comment by marcelofabri** [comment]
@ileitch even Xcode loses itself on your latest example 😅 

anyway, I opened #1635 for the trailing closure indentation, because I think that's another issue.
